examples/plot_unicode.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Únîcödè support in documentation string
+=======================================
+
+In this example we include Unicode characters that are so required this days.✓
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+N = 50
+x = np.linspace(-1,2,50)
```

Comment:
I am one who thinks that Unicode in source code is a bad thing. The reason is that Unicode characters are characters that not everybody can read or type. In a sens, not supporting it forces people not to use it, as they are never something necessary for an example. 

Sent from my phone. Please forgive brevity and mis spelling



On Feb 17, 2015, 08:46, at 08:46, "Loïc Estève" <notifications@github.com> wrote:
>> +=======================================
>> +Únîcödè support in documentation string
>> +=======================================
>> +
>> +In this example we include Unicode characters that are so required
>this days.✓
>> +"""
>> +
>> +# Code source: Óscar Nájera
>> +# License: BSD 3 clause
>> +
>> +import numpy as np
>> +import matplotlib.pyplot as plt
>> +
>> +
>> +N = 50
>> +x = np.linspace(-1,2,50)
>
>Another PEP8 here
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/sphinx-gallery/sphinx-gallery/pull/19/files#r24797986